sdk/cosmosdb/cosmos/src/client/Item/Items.ts

```diff
@@ -373,4 +383,68 @@ export class Items {
       ref
     );
   }
+
+  public async bulk(
+    operations: Operation[],
+    options?: RequestOptions
+  ): Promise<OperationResponse[]> {
+    const {
+      resources: partitionKeyRanges,
+    } = await this.container.readPartitionKeyRanges().fetchAll();
```

Comment:
I assume this returns from a client side cache and doesn't always make a service call?

Reply:
yeah, it should, cc @southpolesteve